crepecake/build.gradle

```diff
@@ -13,9 +13,25 @@ repositories {
     mavenCentral()
 }
 
+sourceSets {
+    integrationTest {
+        java {
+            compileClasspath += main.output + test.output
+            runtimeClasspath += main.output + test.output
+            srcDir file('src/integration-test/java')
+        }
+        resources.srcDir file('src/integration-test/resources')
+    }
+}
+
+configurations {
+    integrationTestCompile.extendsFrom testCompile
+    integrationTestRuntime.extendsFrom testRuntime
+}
+
 dependencies {
     compile gradleApi()
-    compile 'com.google.http-client:google-http-client:1.23.0'
+    compile 'com.google.http-client:google-http-client-jackson2:1.23.0'
```

Comment:
we don't need http-client any more?

Reply:
It seems that -jackson2 comes with the core `google-http-client` artifact.

Reply:
I'm not sure how gradle works here. In google3/blaze we try to be explicit about dependencies. That is, if we directly reference classes from library A and library B, we include both libraries A and B in our BUILD file, even if B pulls in A transitively. However if we reference B which depends on A, but do not explicitly reference classes from A, then we don't include A in our dependency list. This makes the build more consistent as dependencies evolve. However perhaps gradle does things differently.

Reply:
I'll add it back it then. @loosebazooka for clarification on what the best practice here is for Gradle

Reply:
I'd agree with Elliotte here